arrows/ocv/image_container.cxx

```diff
@@ -36,13 +36,30 @@
 #include "image_container.h"
 
 #include <arrows/ocv/mat_image_memory.h>
+#include <opencv2/imgproc/imgproc.hpp>
 
 using namespace kwiver::vital;
 
 namespace kwiver {
 namespace arrows {
 namespace ocv {
 
+image_container
+::image_container(const cv::Mat& d, ColorMode cm)
+  : data_(d)
+{
+  if(cm != RGB)
+  {
+    if ( data_.channels() == 3 )
+    {
+      cv::cvtColor(data_, data_, CV_BGR2RGB);
```

Comment:
Does this convert the color in place?  This could be problematic if the input matrix `d` is shallowed copied to `data_` and then the color of `data_` is converted in place.